datafusion/proto/proto/datafusion.proto

```diff
@@ -480,7 +488,8 @@ message Not {
 
 message AliasNode {
   LogicalExprNode expr = 1;
-  string alias = 2;
+  repeated OwnedTableReference relation = 2;
+  string alias = 3;
```

Comment:
```suggestion
  string alias = 2;
  repeated OwnedTableReference relation = 3;
 ```
Reusing old tag numbers may be dangerous.

References:
- [Don’t Re-use a Tag Number](https://protobuf.dev/programming-guides/dos-donts/#reuse-number)
- [Consequences of Reusing Field Numbers 
](https://protobuf.dev/programming-guides/proto3/#consequences)

Reply:
Good point, committed suggestion.